### PR TITLE
55834 - MOVE/DAS2 : Moves are not correctly displayed considering DAS…

### PR DESCRIPTION
**Pull request by alg-axelor**
…2 settings

### CODE REVIEW

**Review by jmi-axelor — APPROVED** [review]

File: axelor-account/src/main/java/com/axelor/apps/account/service/PartnerAccountService.java [hunk 1]
```diff
@@ -31,4 +41,13 @@ public String getDefaultSpecificTaxNote(Partner partner) {
 
     return fiscalPosition.getCustomerSpecificNoteText();
   }
+
+  @Transactional
```
Comment (gdu-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/service/PartnerAccountService.java:45]: We can't update all partner MoveLines like that. We can have serveral records. Period and JPA cache is not managed, we can't update some old move lines.

File: axelor-base/src/main/resources/views/Partner.xml [hunk 2]
```diff
@@ -991,6 +990,14 @@
     <action name="action-partner-attrs-set-canselect"/>
   </action-group>
 
+  <action-group name="action-partner-group-onsave">
+    <action name="action-partner-iban-check-on-save"/>
+    <action name="save"/>
+    <action name="action-partner-method-set-user-partner"/>
+    <action name="action-partner-method-update-move-line-das2-activity"
+      if="__config__.app.isApp('account')"/>
```
Comment (gdu-axelor) [inline axelor-base/src/main/resources/views/Partner.xml:998]: We can't run huge method on onSave event

File: axelor-account/src/main/java/com/axelor/apps/account/service/AccountService.java [hunk 3]
```diff
@@ -207,4 +211,13 @@ protected Account desactivate(Account account) {
...
   }
+
+  @Transactional
+  public void updateMoveLineServiceType(Account account) {
+    moveLineRepository
+        .all()
+        .filter("self.account = :account")
+        .bind("account", account)
+        .update("serviceType", account.getServiceType());
+  }
```
Comment (gdu-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/service/AccountService.java:222]: We can't update all account MoveLines like that. We can have serveral records. Period and JPA cache is not managed, we can't update some old move lines.

File: axelor-account/src/main/resources/views/Account.xml [hunk 4]
```diff
@@ -207,6 +206,12 @@
     <action name="action-attrs-account-account-status-show-title"/>
   </action-group>
 
+  <action-group name="action-account-group-onSave">
```
Comment (gdu-axelor) [inline axelor-account/src/main/resources/views/Account.xml:209]: We can't run huge method on onSave event

**Review by gdu-axelor — CHANGES_REQUESTED** [review]

**Pull request closed without merging**